### PR TITLE
Make some function arguments const in libpacemaker and libpe_status

### PR DESCRIPTION
**Pull request by kgaillot**
This is not comprehensive but focuses on functions that will be needed for similar changes in daemons

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , here's another low-priority review when you get a chance

File: include/pcmki/pcmki_transition.h [hunk 1]
```diff
@@ -153,8 +153,8 @@ void pcmk__free_graph(pcmk__graph_t *graph);
 const char *pcmk__graph_status2text(enum pcmk__graph_status state);
 void pcmk__log_graph(unsigned int log_level, pcmk__graph_t *graph);
 void pcmk__log_graph_action(int log_level, pcmk__graph_action_t *action);
-lrmd_event_data_t *pcmk__event_from_graph_action(xmlNode *resource,
-                                                 pcmk__graph_action_t *action,
+lrmd_event_data_t *pcmk__event_from_graph_action(const xmlNode *resource,
+                                                 const pcmk__graph_action_t *action,
```
Comment (nrwahl2) [inline include/pcmki/pcmki_transition.h:157]: over 80 if you care

**Comment by nrwahl2** [comment]
Bump copyright on status.h. Otherwise looks good to me.

**Comment by kgaillot** [comment]
thanks, done

**Pull request merged**